### DISCUSSION TITLE
No players joined the public session I created

### DISCUSSION THREAD

**Discussion by iwanttodo1t** [YimMenu/YimMenu#3540] [Q&A]
It doesn't seem to be a network issue.
It was normal before, but then the number of players joining gradually decreased, until now there are no players joining at all.
Are there some rules that we are not aware of?

**Comment by infinity-stack** [comment]
probably because it's a new public session, i think you can do multiplex session or something where network is and press spoofing

**Reply by iwanttodo1t** [reply]
I have found the reason
